device/celestica/x86_64-cel_seastone-r0/sonic_platform/chassis.py

```diff
@@ -301,3 +300,60 @@ def get_status(self):
     def get_thermal_manager(self):
         from .thermal_manager import ThermalManager
         return ThermalManager
+
+    def get_position_in_parent(self):
+        """
+        Retrieves 1-based relative physical position in parent device. If the agent cannot determine the parent-relative position
+        for some reason, or if the associated value of entPhysicalContainedIn is '0', then the value '-1' is returned
+        Returns:
+            integer: The 1-based relative physical position in parent device or -1 if cannot determine the position
+        """
+        return -1
+
+    def is_replaceable(self):
+        """
+        Indicate whether this device is replaceable.
+        Returns:
+            bool: True if it is replaceable.
+        """
+        return False
+
+    def set_status_led(self, color):
+        """
+        Sets the state of the PSU status LED
+        Args:
+            color: A string representing the color with which to set the PSU status LED
+                   Note: Only support green and off
+        Returns:
+            bool: True if status LED state is set successfully, False if not
+        """
+
+        set_status_str = {
+            self.STATUS_LED_COLOR_GREEN: '1',
+            self.STATUS_LED_COLOR_OFF: '0'
+        }.get(color, None)
+
+        if not set_status_str:
+            return False
+
+        try:
+            with open(self.stat_led_path, 'w') as file:
+                file.write(set_status_str)
```

Comment:
Can you use the api helper function to write the file?

Reply:
ok, this function we could use api helper

Reply:
ok